src/helper/service.js

```diff
@@ -0,0 +1,28 @@
+const path = require("path");
+const bcrypt = require("bcryptjs");
+const userList = require("../user.json");
+const fs = require("fs");
+
+const addUser = async (user) => {
+  // Add the new user
+  user.password = await bcrypt.hash(user.password, 3);
+  userList.Users.push(user);
+  let writepath = path.join(__dirname, "..", "user.json");
+  fs.writeFileSync(writepath, JSON.stringify(userList), {
+    encoding: "utf-8",
+    flag: "w",
+  });
+  return user;
+};
+
+//Update user preference
+const updatePrefernce = async (user) => {
```

Comment:
the function parameter name is misleading as you are passing the user list to this function too